### ISSUE TITLE
Boot stuck

### ISSUE THREAD

**Issue by wanyanjiabin** [ThizThizzyDizzy/tree-feller#47]
Hello, after I use your plugin, when I start the server, there is a high probability of being stuck in the loading process. After removing the plugin, the startup is smooth.
Because the server is located in China, it may be caused by network problems. Can you join the network connection time limit? If can't connect, skip it to avoid getting stuck.

The server is paper1.19

**Comment by ThizThizzyDizzy** [comment]
Ohh, I didn't even think about that.
It *should* time out and continue startup if it can't connect, but either way downloading the patrons list doesn't need to hold up startup; (that's probably what it's doing)

**Comment by wanyanjiabin** [comment]
About 70% of the probability will be stuck in the startup process, sorry, maybe few Chinese players use this plug-in, so there has been no feedback. After mc1.19, only your plug-in can be used. So, you can Fix it. Thanks a lot

**Comment by wanyanjiabin** [comment]
About 70% of the probability will be stuck in the startup process, sorry, maybe few Chinese players use this plug-in, so there has been no feedback. After mc1.19, only your plug-in can be used. So..... can you Fix it. Thanks a lot ^_^

**Comment by ThizThizzyDizzy** [comment]
Yep, I'll push out a hotfix for it when I get home ^-^ (probably in about 8 hours)

**Comment by wanyanjiabin** [comment]
thank you very much!!! ^.^

**Comment by ThizThizzyDizzy** [comment]
Forgot it yesterday, but I finally got to it- fixed in b052dcac6219152565be5d77169dc839771579dc, hotfix v1.19.3 coming shortly
